templates/interfaces.j2

```diff
@@ -1,49 +1,130 @@
-# vim: sw=4 ts=4 ft=jinja
 # {{ ansible_managed }}
 
+{% if network_management_pre_up is defined %}
+pre-up {{ network_management_pre_up }}
+
+{% endif %}
+
 auto lo
 iface lo inet loopback
 
-{% for interface in bridges %}
-{% for port in interface.ports | default({}) %}
+{% if network_management_clear_bridges %}
```

Comment:
Check if bridges are defined. This is not skipped, because it is not inside of a for body.

Reply:
Fixed